lib/SIL/Utils/OwnershipUtils.cpp

```diff
@@ -377,19 +377,26 @@ bool swift::findExtendedUsesOfSimpleBorrowedValue(
   return true;
 }
 
+// TODO: refactor this with SSAPrunedLiveness::computeLiveness.
 bool swift::findUsesOfSimpleValue(SILValue value,
                                   SmallVectorImpl<Operand *> *usePoints) {
   for (auto *use : value->getUses()) {
-    if (use->getOperandOwnership() == OperandOwnership::Borrow) {
+    switch (use->getOperandOwnership()) {
```

Comment:
Do we need the same change in LexicalDestroyFolding's findBorroweeUsage?

Reply:
👍 I'll need to figure out a test case for that one